Bluepill-runner/BluepillRunnerTests/BPReportCollectorTests.m

```diff
@@ -31,11 +31,11 @@ - (void)testCollectReportsFromPath {
     NSString *path = [[NSBundle bundleForClass:[self class]] resourcePath];
     NSString *outputPath = [path stringByAppendingPathComponent:@"result.xml"];
     [BPReportCollector collectReportsFromPath:path onReportCollected:^(NSURL *fileUrl) {
-        NSError *error;
-        NSFileManager *fm = [NSFileManager new];
-        [fm removeItemAtURL:fileUrl error:&error];
-        XCTAssertNil(error);
-    }  outputAtPath:outputPath];
+//        NSError *error;
```

Comment:
Wouldn't it be simpler to pass `nil` in `onReportCollected` here? Seems weird to have a closure with just commented code.

Reply:
Removed this